src/views/docs/en/reference/project-manifest/tables-streams.md

```diff
@@ -0,0 +1,132 @@
+---
+title: '@tables-streams'
+category: app.arc
+description: Define DynamoDB tables with streaming changes
+---
+
+Define Lambda functions for streaming changes to DynamoDB tables. Respond to `insert`, `update`, and `destroy` events with a handler function.
+
+> ⚠️  Unfortunately, "dynalite" (used under the hood in Architect [Sandbox](../cli/sandbox)) doesn't support streams, so emulation isn't yet supported for local development.
+
+## Recommended Resources
+
+[AWS DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.Lambda.html)
+
+## Syntax
+
+- Name
+  - Lowercase alphanumeric string
+  - Must match a `@tables` name
+
+Table streams can use more verbose configuration to allow for [custom source paths](../../guides/developer-experience/custom-source-paths) and names in your project. Optionally provide a `name` and/or `src` for each table stream.
```

Comment:
Might be worth mentioning this is how you can specify multiple distinct table stream lambdas for a single table.